lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -113,22 +113,6 @@ describe('CategoryRenderer', () => {
     category.description = prevDesc;
   });
 
-  it('renders audits with debugString as failed', () => {
```

Comment:
do we have a test for this that handles rendering of the new modes? e.g. I don't see an error one

Reply:
tbh i'm not sure. :)

i'd like to do a followup once we have the 3 PRs landed (this one, scoreDisplayMode, and debugString -> warnings/errors/explanations) where we can try all the possibilities.

Reply:
> i'd like to do a followup 

add to #5008?